src/Microsoft.AspNetCore.Razor.VSCode/src/IRazorProjectConfiguration.ts

```diff
@@ -0,0 +1,17 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import * as vscode from 'vscode';
+
+export interface IRazorProjectConfiguration {
+    readonly path: string;
+    readonly uri: vscode.Uri;
+    readonly projectPath: string;
+    readonly projectUri: vscode.Uri;
+    readonly tagHelpers: any[];
+    readonly configuration: any;
```

Comment:
Do we only use this `configuration` data to pass through to the language server without inspecting it on the JS side? If so, totally reasonable to type it as `any`. If we do need to process it within TypeScript though it would be nice to have a type definition for it.

Reply:
Mostly pass through. The only other thing that will be inspecting it will be the telemetry bits to report on project types.

Reply:
I'll address that in the telemetry PR.